apache-rat-core/src/test/java/org/apache/rat/document/impl/guesser/NoteGuesserTest.java

```diff
@@ -34,34 +39,24 @@ public void testMatches() {
         assertTrue(NoteGuesser.isNote(new TestingDocument("NOTICE.txt")));
         assertTrue(NoteGuesser.isNote(new TestingDocument("README")));
         assertTrue(NoteGuesser.isNote(new TestingDocument("README.txt")));
-    }
+        assertTrue(NoteGuesser.isNote(new TestingDocument("src/test/DEPENDENCIES")));
+        assertTrue(NoteGuesser.isNote(new TestingDocument("src/test/LICENSE")));
+        assertTrue(NoteGuesser.isNote(new TestingDocument("src/test/LICENSE.txt")));
+        assertTrue(NoteGuesser.isNote(new TestingDocument("src/test/NOTICE")));
+        assertTrue(NoteGuesser.isNote(new TestingDocument("src/test/NOTICE.txt")));
+        assertTrue(NoteGuesser.isNote(new TestingDocument("src/test/README")));
+        assertTrue(NoteGuesser.isNote(new TestingDocument("src/test/README.txt")));
 
-    @Test
-    public void isNote() {
-        assertTrue(NoteGuesser.isNote("DEPENDENCIES"));
-        assertTrue(NoteGuesser.isNote("LICENSE"));
-        assertTrue(NoteGuesser.isNote("LICENSE.txt"));
-        assertTrue(NoteGuesser.isNote("NOTICE"));
-        assertTrue(NoteGuesser.isNote("NOTICE.txt"));
-        assertTrue(NoteGuesser.isNote("README"));
-        assertTrue(NoteGuesser.isNote("README.txt"));
     }
-    
-    @Test
-    public void isNoteWithPath() {
-        assertTrue(NoteGuesser.isNote("src/test/DEPENDENCIES"));
-        assertTrue(NoteGuesser.isNote("src/test/LICENSE"));
-        assertTrue(NoteGuesser.isNote("src/test/LICENSE.txt"));
-        assertTrue(NoteGuesser.isNote("src/test/NOTICE"));
-        assertTrue(NoteGuesser.isNote("src/test/NOTICE.txt"));
-        assertTrue(NoteGuesser.isNote("src/test/README"));
-        assertTrue(NoteGuesser.isNote("src/test/README.txt"));
-        assertTrue(NoteGuesser.isNote("src\\test\\DEPENDENCIES"));
-        assertTrue(NoteGuesser.isNote("src\\test\\LICENSE"));
-        assertTrue(NoteGuesser.isNote("src\\test\\LICENSE.txt"));
-        assertTrue(NoteGuesser.isNote("src\\test\\NOTICE"));
-        assertTrue(NoteGuesser.isNote("src\\test\\NOTICE.txt"));
-        assertTrue(NoteGuesser.isNote("src\\test\\README"));
-        assertTrue(NoteGuesser.isNote("src\\test\\README.txt"));
+
+    @Disabled("RAT-390: old tests that do not work anymore")
```

Comment:
@Claudenw shouldn't these cases work as the normalisation is supposed to remove path separators? I just reformatted the commented-out stuff and added it here as the tests with backslashes do not recognise the files as a note.